target/linux/ath79/dts/qca9531_comfast_cf-e355ac-v2.dts

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "comfast,cf-e355ac-v2", "qca,qca9531";
+	model = "COMFAST CF-E355AC V2";
+
+	aliases {
+		serial0 = &uart;
+		label-mac-device = &eth1;
+		led-boot = &led_lan;
+		led-failsafe = &led_lan;
+		led-upgrade = &led_lan;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		led_lan: lan {
+			label = "green:lan";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
```

Comment:
redundant empty line :)
also you might consider adding `default-state = "on";` to `led_lan`, in case there is a ~2 seconds gap where the led is turned off, right before the failsafe blinking pattern starts.

Reply:
Since this is a hijacked LAN LED, I'd rather not put the default-state here, so we have some chance to see the actual LED behavior from bootloader ...